src/overlays/actors/ovl_Obj_Kibako/z_obj_kibako.c

```diff
@@ -20,42 +39,375 @@ const ActorInit Obj_Kibako_InitVars = {
     (ActorFunc)ObjKibako_Update,
     (ActorFunc)NULL,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Kibako_0x809261B0/func_809261B0.asm")
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_PLAYER,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00400000, 0x00, 0x02 },
+        { 0x058BC79C, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NONE,
+        BUMP_ON,
+        OCELEM_ON,
+    },
+    { 15, 30, 0, { 0, 0, 0 } },
+};
+
+static s16 sObjectBankTable[] = { GAMEPLAY_DANGEON_KEEP, OBJECT_KIBAKO };
+
+static Gfx* sParticleDisplayLists[] = { 0x05007980, 0x06001A70 };
+
+static Gfx* sMainDisplayLists[] = { 0x05007890, 0x06001180 };
```

Comment:
should probably just be sDlists, is there a "sub" dlist anywhere that would make this be the "main" dlist?